.github/workflows/integration-tests.yml

```diff
@@ -21,8 +9,8 @@ on:
         required: true
         default: dev
         options:
-        - dev
-        - uat
+          - dev
+          - uat
```

Comment:
is this indentation correct? if so, [this file](https://github.com/pagopa/pdv-ms-tokenizer/blob/e23712244014f17989501d6388a7014f2baa532d/.github/workflows/integration-tests.yml#L12C14-L12C14) should be edited in the same way

Reply:
I think both are correct, but for uniformity I'll set the same indentation